ledger/accountdb.go

```diff
@@ -1737,6 +1771,26 @@ func (qs *accountsDbQueries) lookupCreator(cidx basics.CreatableIndex, ctype bas
 	return
 }
 
+func (qs *accountsDbQueries) lookupResources(addr basics.Address, aidx basics.CreatableIndex, ctype basics.CreatableType) (data persistedResourcesData, err error) {
+	data.aidx = aidx
+	data.rtype = ctype
```

Comment:
I would fill these up only in the case of a success ( i.e. err == nil ).
In the case of a failure, it wouldn't help if we were to fill these up.